encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -25,14 +25,17 @@
 package main
 
 import (
+	"flag"
 	"path/filepath"
 	"strings"
 
 	"go.uber.org/thriftrw/plugin"
 	"go.uber.org/thriftrw/plugin/api"
 )
 
-const serverTemplate = `
+var _uberfx = flag.Bool("uberfx", false, "generate code for UberFx with fx.Context")
```

Comment:
Yikes - I think we don't actually want an `-uberfx` flag here.

Was hoping that you could pass in a context path into the plugin instead - something generic that could be used by other libraries as well.